src/js/letters/containers/LettersApp.jsx

```diff
@@ -16,6 +17,7 @@ export function AppContent({ children, isDataAvailable }) {
         If you think you should be able to access this information, please call the Vets.gov Help Desk at <a href="tel:855-574-7286">1-855-574-7286</a>, TTY: <a href="tel:18008778339">1-800-877-8339</a>, Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET).
       </h4>
     );
+    Raven.captureException(new Error('vets_letters_user_unregistered'));
```

Comment:
Would appreciate some 👀  and thought on this - I'm specifically concerned about potential to log multiple times as the component re-renders, but don't think this will be an issue because of the way the `isDataAvailable` prop is passed from `RequiredLoginView`

Reply:
Agreed that this isn't an issue since it seems to only re-render upon page refresh.